Tests/ImmutableDataTests/StoreTests.swift

```diff
@@ -158,8 +157,9 @@ extension StoreTests {
 
 extension StoreTests {
   @Test func dispatchThunkThrowsError() async throws {
-    self.thunk.returnError = Error()
-    
+    let error = Error()
```

Comment:
I went with a local constant that's known to be non-optional to make the expectation below more directly tell that we don't expect the error to be non-nil whenever the thunk's `returnError` is non-nil (for which the reader needs to find the conditions), but that it's always going to be a local constant set-up in this test case.